packages/dialog/README.md

```diff
@@ -0,0 +1,397 @@
+| `defaultAction`         | `string`  | _Default: 'close'_ – Action to be emitted with the `closing` and `closed` events when `<mwc-dialog>.open` is toggled (see [actions section](#actions)).
+| `actionAttribute`       | `string`  | _Default: 'dialog-action'_ – Attribute to read in light dom of dialog for closing action value (see [actions section](#actions)).
+| `initialFocusAttribute` | `string`  | _Default: 'dialog-initial-focus'_ – Attribute to search for in light dom for initial focus on dialog open.
+
+### Methods
+
+| Name     | Description
+| -------- | -------------
+| `forceLayout() => void` | Forces dialog to relayout (animation frame time). May be required if dialog size is incorrect or if stacked layout has not been triggered correctly.
+| `focus() => void` | Focuses on the initial focus element if defined (see [focus section](#focus)).
+| `blur() => void` | Blurs the active element.
+
+### Listeners
+| Event Name          | Target       | Description
+| ------------------- | ------------ | -----------
+| `click`             | root element | Detects if clicked target is a dialog action.
+| `resize`            | `window `    | Performs dialog layout (passive).
+| `orientationchange` | `window`     | Performs dialog layout (passive).
+| `keydown`           | `mwc-dialog` | Listens for the enter key to click the default button (passive).
+| `keydown`           | `document`   | Listens for the escape key to close the dialog (see [`escapeKeyAction`](#properties)).
+
+### Events
+
+| Event Name | Target       | Detail             | Description
+| ---------- | ------------ | ------------------ | -----------
+| `opening`  | `mwc-dialog` | `{}`               | Fired when the dialog is beginning to open.
+| `opened`   | `mwc-dialog` | `{}`               | Fired once the dialog is finished opening (after animation).
+| `closing`  | `mwc-dialog` | `{action: string}` | Fired when the dialog is is beginning to close. Detail is the action that closed the dialog (see [actions section](#actions)).
+| `closed`   | `mwc-dialog` | `{action: string}` | Fired once the dialog is finished closing (after animation). Detail is the action that closed the dialog (see [actions section](#actions)).
+
+### CSS Custom Properties
+
+Inherits CSS Custom properties from:
+* [`mwc-ripple`](https://github.com/material-components/material-components-web-components/tree/master/packages/ripple)
+* [`mwc-notched-outline`](https://github.com/material-components/material-components-web-components/tree/master/packages/notched-outline).
+* [`mwc-icon`](https://github.com/material-components/material-components-web-components/tree/master/packages/icon)
+
+| Name                                | Default               | Description
+| ----------------------------------- | --------------------- |------------
+| `--mdc-theme-surface`               | `#fff`                | Color of the dialog surface's background.
```

Comment:
Color swatches?

Reply:
[MDC Theme](https://github.com/material-components/material-components-web/tree/master/packages/mdc-theme) has a concept of "surface":

```
--mdc-theme-surface The theme surface color
```

Reply:
Sure. Do we want to have the little color swatch next the the defaults like in the other READMEs?